deltachat-ios/Controller/ChatViewController.swift

```diff
@@ -472,6 +481,20 @@ class ChatViewController: MessagesViewController {
             super.collectionView(collectionView, performAction: action, forItemAt: indexPath, withSender: sender)
         }
     }
+
+    private func askToDeleteMessage(id: Int) {
+        let alert = UIAlertController(title: String.localized("delete_message_ask"),
```

Comment:
i think we should re-use `String.localizedStringWithFormat(String.localized("ask_delete_messages"), 1)`. already available in all languages, less work for translators and gives hint that the message is also deleted from the server.

Reply:
fixed.